scripts/generate_schemas.py

```diff
@@ -317,8 +366,9 @@ def main():
     parser.add_argument(
         "version",
         help="The tagged git release to process, e.g., v0.5.1")
-    # TODO is this the right approach? Maybe we should be noisy be
```

Comment:
I'm not sure we should delete this - should we be noisy or quiet by default here? It's not obvious to me what's best. It was actually quite useful to have the output from the java process, as this pointed out a bug in the schema definitions. (See ga4gh/schemas#195 for this issue). It would be good to discuss the best approach here anyway I think.